pkgs/io_file/lib/src/file_system.dart

```diff
@@ -30,6 +30,18 @@ class WriteMode {
 
 /// An abstract representation of a file system.
 base class FileSystem {
```

Comment:
Make this class abstract?

Reply:
We've been discussing this at https://github.com/dart-lang/labs/pull/202

I'll leave it unchanged here and hopefully we can come to a conclusion in that PR ;-)